index.js

```diff
@@ -11,28 +11,75 @@ const nextButton = document.querySelector('[data-js="button-next"]');
 const pagination = document.querySelector('[data-js="pagination"]');
 
 // States
-const maxPage = 1;
-const page = 1;
+let maxPage = 0;
+let page = 1;
 const searchQuery = "";
 
+fetchCharacters("character", page);
+
+// Pagination - final solution
+nextButton.addEventListener("click", () => {
+  page++;
+  prevButton.removeAttribute("disabled");
+  if (page > maxPage) {
+    page = maxPage;
+    nextButton.setAttribute("disabled", "disabled");
+  } else {
+    nextButton.removeAttribute("disabled");
+    fetchCharacters("character", page);
+  }
+  pagination.textContent = `${page} / ${maxPage}`;
+});
```

Comment:
If you increase the page number only when `page` is smaller than `maxPage`, you do not need to disable the buttons. Right now, you always increase the page number when the button is clicked.

```suggestion
nextButton.addEventListener("click", () => {
  if (page < maxPage) {
    page++;
    fetchCharacters("character", page);
  })
```

The text content of your pagination you can set inside your fetch function, so you only need to do that once.